### ISSUE TITLE
CRI-O Can't get running containers stats 

### ISSUE THREAD

**Issue by Nefelim4ag** [cri-o/cri-o#3501]
```
[pid  1048] newfstatat(AT_FDCWD, "/sys/fs/cgroup/cpu/kubepods-burstable-pod7c05aa2f1e41abcba30a9e63ee8d5384.slice", 0xc427e709f8, 0) = -1 ENOENT (No such file or directory)
[pid  1048] newfstatat(AT_FDCWD, "/sys/fs/cgroup/blkio/kubepods-burstable-pod7c05aa2f1e41abcba30a9e63ee8d5384.slice", 0xc427e71968, 0) = -1 ENOENT (No such file or directory)
[pid  1048] newfstatat(AT_FDCWD, "/sys/fs/cgroup/memory/kubepods-burstable-pod77f2a716b640e00e11d5a11f3cfdf917.slice", 0xc427e6e858, 0) = -1 ENOENT (No such file or directory)
[pid  1048] newfstatat(AT_FDCWD, "/sys/fs/cgroup/cpu/kubepods-burstable-podaeb8c19b6aa2a77d07cd8100a669d7e5.slice", 0xc427e6f7c8, 0) = -1 ENOENT (No such file or directory)
[pid  1048] newfstatat(AT_FDCWD, "/sys/fs/cgroup/pids/kubepods-burstable-pod7c05aa2f1e41abcba30a9e63ee8d5384.slice", 0xc4206b2858, 0) = -1 ENOENT (No such file or directory)
[pid  1048] newfstatat(AT_FDCWD, "/sys/fs/cgroup/cpu/kubepods-burstable-podccf563b788dd2097a61843086e04d5dd.slice", 0xc4206b3a38, 0) = -1 ENOENT (No such file or directory)

```

CRI-O by some reason try get CPU stats from wrong path

Right path: `/sys/fs/cgroup/cpu/kubepods.slice/kubepods-burstable.slice/`

May be i do something wrong?

K8s 1.16.4 & cri-o 1.17.1

**Comment by Nefelim4ag** [comment]
https://github.com/cri-o/cri-o/commit/f2d3397f4f66ea75cdfce1d34bdf8f37276eaa37

**Comment by saschagrunert** [comment]
Hey @Nefelim4ag do you have the kubelet and CRI-O both configured as systemd cgroup manager?

**Comment by Nefelim4ag** [comment]
Fixed in 1.17.2 CRI-O
@saschagrunert yep
